src/compinterpret.py

```diff
@@ -59,6 +62,11 @@ def __init__(self) -> None:
             '}': '}'
         }    
 
+        regional_currency = locale.localeconv()['currency_symbol']
+        if regional_currency == '':
+            # Americentrisim, baby 😎🦅🔫🔫🦅🦅🦅🔫🦅 🦅🔫🔫🔫🦅🔫🦅🔫 🦅🔫🔫🔫🦅🦅🔫🔫 🦅🔫🔫🦅🔫🦅🦅🦅 🦅🦅🔫🦅🦅🦅🦅🦅 🦅🔫🔫🦅🦅🔫🦅🦅 🦅🔫🔫🦅🔫🦅🦅🔫 🦅🔫🔫🦅🦅🔫🦅🦅 🦅🦅🔫🦅🦅🦅🦅🦅 🦅🦅🔫🔫🔫🦅🦅🔫 🦅🦅🔫🦅🔫🔫🔫🔫 🦅🦅🔫🔫🦅🦅🦅🔫 🦅🦅🔫🔫🦅🦅🦅🔫😎
+            regional_currency = '$'
```

Comment:
I think using the character **`¤`** might be better because it's literally its purpose (the default currency symbol)

Reply:
Oh neat, I didn't know that was a thing- will change in a sec